README.md

```diff
@@ -50,6 +50,15 @@ cd cgt
 cargo install --path "."
 ```
 
+### Running inside a container
+
+An alternative, if you are having trouble with the above, is to use the CELEBRIMBOR docker
+container. If you are comfortable running commands inside docker containers and mounting
+your external files, the whole pipeline is in the container available by running:
+```
+docker pull samhorsfield96/celebrimbor:main
+```
```

Comment:
I think it would help to give one example command of how to use it as in e.g. the excellent ggcaller installation documentation:
```
cd test && docker run --rm -it -v $(pwd):/workdir samhorsfield96/ggcaller:latest ggcaller --refs pneumo_CL_group2.txt
```